scvid/data/sampler.py

```diff
@@ -0,0 +1,63 @@
+from typing import Iterator, Sequence
+
+import torch
+from torch.utils.data.sampler import Sampler
+
+
+class DADCSampler(Sampler):
```

Comment:
When you implement multi GPU, do you plan to keep this class as well? If so, perhaps call it `DistributedAnnDataCollectionSingleConsumerSampler`? :D